examples/maxmod/streaming/source/main.c

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: CC0-1.0
+//
+// SPDX-FileContributor: Michele Di Giorgio, 2024
+//
+// Example usage of maxmod streaming. The example loads a WAV file and streams
+// the data through maxmod's utilities.
+//
+// Audio track credits:
+//  https://www.looperman.com/loops/detail/222561/kingmike16-120bpm-techno-synth-loop
+
+#include <stdio.h>
+
+#include <filesystem.h>
+#include <maxmod9.h>
+#include <nds.h>
+
+#define WAV_FILENAME "nitro:/fstest/looperman.wav"
```

Comment:
Just a small detail, move the wav file one level up (remove the `fstest` folder).

Reply:
Done